R/calculate_geom_positions.R

```diff
@@ -154,22 +154,20 @@ calculate_geom_positions <- function(
       ungroup()
   })
 
-  test_processor <- geom_data_processor("image", function(dat) {
-    dat
-  })
-
   # gather image data
   img_data <- geom_data_processor("image", function(dat) {
     # If the location of the image is not provided using the "value"
     # column in the data, construct it from the directory and filename
-    if (!is.null(dat$extension)) {
+    print(dat$extension)
+    if (dat %has_name% "directory") {
       dat$value <- paste0(dat$directory, dat$value, ".", dat$extension)
-    }
+    } 
```

Comment:
Print should probably be removed. Also, it might be better to process the directory and the extension separately, and to support having NA values in the column (for some reason).

```suggestion
    if (dat %has_name% "directory") {
      dat$value <- ifelse(is.na(dat$value) | is.na(dat$directory), dat$value, paste0(dat$directory, "/", dat$value))
    }
    if (dat %has_name% "extension") {
      dat$value <- ifelse(is.na(dat$value) | is.na(dat$extension), dat$value, paste0(dat$value, ".", dat$extension))
    }
```